thirdparty/chart-auth-svc/templates/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if .Values.enabled -}}
+apiVersion: {{ template "apiVersion.Deployment" . }}
+kind: Deployment
+metadata:
+  name: {{ include "auth-svc.name" . }}
+  labels:
+    {{- include "auth-svc.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+  selector:
+    matchLabels:
+      {{- include "auth-svc.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "auth-svc.selectorLabels" . | nindent 8 }}
```

Comment:
This should be `labels` not `selectorLabels`, btw this is also something that needs to be fixed in the ml-operator chart.

`selectorLabels` should only be used for selectors ;D